src/lib/router.ts

```diff
@@ -39,6 +39,29 @@ router.post('/matches', async (request, response, next) => {
   }
 });
 
+router.get('/matches/:matchId', async (request, response, next) => {
+  try {
+    const { matchId } = request.params;
```

Comment:
isn`t better to validate the match id, to avoid mongoserver errors? 

````js
const matchIdIsInvalid = !ObjectId.isValid(matchId);

if (matchIdIsInvalid) {
  response.status(400).send('Invalid match id');
  return;
}